PHPCI/Helper/Lang.php

```diff
@@ -168,4 +168,20 @@ protected static function loadAvailableLanguages()
             }
         }
     }
+
+    /**
+     * Create a time tag for localization.
```

Comment:
Empty line between function description, params and from params and return value.